lib/masamune/transform/deduplicate_dimension.psql.erb

```diff
@@ -20,6 +20,17 @@
 --  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 --  THE SOFTWARE.
 
+WITH consolidated AS (
+  SELECT
+    <%- target.insert_view_values(coalesce: true).each do |value| -%>
+    <%= value %><%= ',' %>
```

Comment:
necessary to `COALESCE` values with their default value *before* attempting to upsert staged table